README.md

```diff
@@ -1,14 +1,21 @@
 # Dash SDK
 
-[![Package Version](https://img.shields.io/github/package-json/v/dashevo/dashjs.svg?&style=flat-square)](https://www.npmjs.org/package/dash)
+[![NPM Version](https://img.shields.io/npm/v/dash)](https://img.shields.io/npm/v/dash)
 [![Build Status](https://img.shields.io/travis/com/dashevo/dashjs.svg?branch=master&style=flat-square)](https://travis-ci.com/dashevo/dashjs)
+[![Release Date](https://img.shields.io/github/release-date/dashevo/dashjs)](https://img.shields.io/github/release-date/dashevo/dashjs)
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
```

Comment:
Also can we consistently use either the default style `flat` or the `flat-square` style for all badges?

Reply:
Yes, let's do that. Do you have a preference?

Reply:
I don't have one, dealer's choice unless @strophy has a strong opinion

Reply:
I prefer `flat` because it is the default. So we can just omit any `style` specification.

Reply:
Fixed